package.json

```diff
@@ -76,6 +76,7 @@
     "chrome-remote-interface": "0.27.1",
     "cordova": "7.1.0",
     "cordova-android": "7.1.4",
+    "cozy-jobs-cli": "1.8.14",
```

Comment:
We should avoid adding a dependency that isn't used anywhere. If it's a dev tool, I would either keep it as a global dependency on my machine, or add the corresponding scripts to the package.json.

Reply:
Hm, I tend to disagree for this, this is a dev tools especially needed for the service, so for me it's relevant to be included in the devDep. It is actually done like this in banks: https://github.com/cozy/cozy-banks/blob/master/package.json#L103

Reply:
In that case it should be used for something in the repo, either in a script, or a documentation. Otherwise the tool is only usable by a single person who happens to know how to use it.

(The other reason why adding the usage somewhere in the repo is important is because if someone in 3 months decides to remove outdated dependencies, this one would get removed because it has no apparent usage.)

Reply:
Fair enough for the documentation :)
Would it be ok in a README in the service [directory](https://github.com/cozy/cozy-drive/tree/master/src/photos/targets/services)?

Reply:
Yes!